fractal-puppy/Engine
# Player weapon can be changed

If you equip a weapon using the inventory the weapon mesh and material will change for the one equipped.

This can be extended for any item, but currently, it is only working with weapons and helmets. Items have two new variables: mesh and material. If an item has a mesh and a material set, once it is equipped using the inventory, the mesh will appear on the player.

The graveyard scene has been updated, now you start without a weapon. A weapond can be found on the floor next to the player.

**To test:**
1. Open the graveyard scene.
2. Hit play and take the weapon on the floor.
3. Equip the weapon using the inventory menu.
4. See if once equipped a mesh of a shovel appears on the hand of the player.
5. Unequip the weapon and see if it disappears.
